src/github/logic.py

```diff
@@ -307,11 +304,36 @@ def maybe_automerge_pull_request_and_rerun_stale_checks(
     return False
 
 
+def maybe_rerun_stale_checks_on_approved_pull_request(
+    pull_request: PullRequest,
+) -> bool:
+    if (
+        SGTM_FEATURE__ALLOW_CHECK_RERUN_ON_APPROVAL
+        and _pull_request_is_open(pull_request)
+        and pull_request.is_approved()
+    ):
+        logger.info(
+            f"PR-{pull_request.id()} is open and approved, maybe rerun stale checks"
+        )
+        return _maybe_rerun_stale_checks(pull_request)
+    logger.info(
+        f"SGTM_FEATURE__ALLOW_CHECK_RERUN_ON_APPROVAL is {SGTM_FEATURE__ALLOW_CHECK_RERUN_ON_APPROVAL}"
```

Comment:
I don't think we log any of the feature flags. The exception being https://github.com/Asana/SGTM/blob/5f83954ace9ab36ae8d336053bfac45ea51c1a38/src/github/logic.py#L326 but this log exists mainly to check the base ref name of the PR. What value does this log line give?

Reply:
was using this for testing but can remove!